tests/core/test_model.py

```diff
@@ -1764,10 +1764,10 @@ def a_model(context):
         path=Path("."),
     )
 
-    assert isinstance(python_model.kind, ViewKind)
+    assert isinstance(python_model.kind, FullKind)
 
     # string kind name specified
-    @model("kind_string", kind="full", columns={'"COL"': "int"})
+    @model("kind_string", kind="external", columns={'"COL"': "int"})
```

Comment:
What's this test? I don't think users suppose to ever set the `external` kind themselves.

Reply:
It was more because `full` was the default, so I needed to do another kind to show that it was actually parsing the value and wasn't just falling back to the default.

I first chose `incremental_by_time_range` but that has other mandatory parameters like `time_column` which I couldn't figure out how to represent in the string, so I settled on `external` because that one is simple.